static/js/create-worklist.js

```diff
@@ -0,0 +1,64 @@
+// logic for adding new article box on button click
+var noOfArticles = 0;
+function addNewArticle() {
+    noOfArticles++;
+    var html = '<div id="article-'+noOfArticles+'"><br/><h3>Article '
+        + noOfArticles +'</h3>Name of Article: <input style="width: 572px;" type="text" ' +
+        'class="article-name" name="article-name" placeholder="Enter name of the article">' +
```

Comment:
Not for now, but I think we'll want this to autocomplete by the way.

https://www.mediawiki.org/wiki/API:Search and https://github.com/LeaVerou/awesomplete are possibly what we need.

Reply:
After the basic functionality is complete, I'll take up this.